ArrtSource/Conversion/UI/ConversionTab.cpp

```diff
@@ -233,6 +233,51 @@ void ArrtAppWindow::UpdateConversionPane()
         ConversionTab->VertexBitangentCombo->setCurrentIndex((int)opt.m_vertexBinormal);
         ConversionTab->TexCoord0Combo->setCurrentIndex((int)opt.m_vertexTexCoord0);
         ConversionTab->TexCoord1Combo->setCurrentIndex((int)opt.m_vertexTexCoord1);
+
+        const auto flags = conv.m_availableOptions;
+
+        ConversionTab->ScalingSpinbox->setVisible(flags & (uint64_t)ConversionOption::UniformScaling);
+        ConversionTab->ScalingSpinboxL->setVisible(flags & (uint64_t)ConversionOption::UniformScaling);
+        ConversionTab->RecenterToOriginCheckbox->setVisible(flags & (uint64_t)ConversionOption::RecenterToOrigin);
+        ConversionTab->RecenterToOriginCheckboxL->setVisible(flags & (uint64_t)ConversionOption::RecenterToOrigin);
+        ConversionTab->DefaultSidednessCombo->setVisible(flags & (uint64_t)ConversionOption::MaterialDefaultSidedness);
+        ConversionTab->DefaultSidednessComboL->setVisible(flags & (uint64_t)ConversionOption::MaterialDefaultSidedness);
+        ConversionTab->GammaToLinearMaterialCombo->setVisible(flags & (uint64_t)ConversionOption::GammaToLinearMaterial);
+        ConversionTab->GammaToLinearMaterialCheckboxL->setVisible(flags & (uint64_t)ConversionOption::GammaToLinearMaterial);
+        ConversionTab->GammaToLinearVertexCombo->setVisible(flags & (uint64_t)ConversionOption::GammaToLinearVertex);
+        ConversionTab->GammaToLinearVertexCheckboxL->setVisible(flags & (uint64_t)ConversionOption::GammaToLinearVertex);
```

Comment:
Is the naming here just inconsistent?
It just differs from the rest, but could of course be right. Just want to double check, that it is the correct thing.

Reply:
It went through several iterations and I didn't have the motivation to rename all the variables again.